doc/rst/language/spec/modules.rst

```diff
@@ -531,6 +638,44 @@ well - in the first example, if module A’s use of module B contains an
 ``except`` or ``only`` list, that list will also limit which of C’s
 symbols are visible to A.
 
+.. _Using_Enums:
+
+Using Enums
++++++++++++
+
+Aside from modules, only enums can be listed as the last portion of a ``use``
+statement's ``module-or-enum-name``.  Doing so enables its constants to be
+accessible without the enum's name as a prefix (see :ref:`Explicit_Naming` for
+how to access its constants normally).
+
+All of the rules stated above apply when using an enum instead of a module.
+Because enums cannot contain use statements, the transitivity description only
+applies when a module uses a module with a public use of an enum.  For example:
```

Comment:
I'm not sure what the "transitivity description" is and this paragraph and the example seem to me to be like stuff that follows from the rest and might not be necessary.

Reply:
I wrote this section before deciding to make public and private use statements an explicit section, so that was more talking about that section in vague terms.  It will probably work better with a link back, but I'm more inclined to just remove it as well

Reply:
Removing it sounds good to me!